### PR TITLE
[#1934]  Add Image Badges

### LINKED ISSUE #1934: [DOCS] Add Badges
### Issue Description

I would like to add interactive badges in a table format, int the header-section of the README documentation. This will enhance the Engagement & interface of the README.

I would like to work on this issue. can you assign this issue to me. @RymMichaut 

### Screenshots or Examples (if applicable)

image

### Proposed Solution (optional)

Use badges to add dynamic & real time updating that will show simple stats.

### Code of Conduct

- [X] I have checked the [existing issues](https://github.com/Avaiga/taipy/issues?q=is%3Aissue+).
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by droidbg**
## What type of PR is this? (check all applicable)
- [X] Documentation

## Description
This PR include changes required in issue #1934 for the README.md file to provide more information to users.

- Added Badges for license, python version, release version
- Added Badges for documentation link, example link, discord link

## Related Tickets & Documents
- Closes #1934

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +9,52 @@
...
+<br>
+<div align="center">
+    <img 
+        src="https://img.shields.io/github/license/Avaiga/taipy?style=plastic&color=ff371a&labelColor=1f1f1f" 
+        alt="GitHub License" 
+        height="20px" 
+    />
+    &nbsp;
+    <img 
+        src="https://img.shields.io/badge/python%20version-3.8-ff371a?style=plastic&labelColor=1f1f1f"
```
Comment (jrobinAV) [inline README.md]: This is incorrect. We don't support 3.8 anymore.

Please add also versions 3.9, 3.10, 3.11, 3.12

Thx

File: README.md [hunk 2]
```diff
@@ -10,6 +9,52 @@
...
+            height="20px" 
+            alt="Explore the docs" 
+        ></a>
+    &nbsp;
+    <a target="_blank" href="https://docs.taipy.io/en/latest/gallery/">
+        <img 
+            src="https://img.shields.io/badge/examples-ff371a?style=plastic&labelColor=1f1f1f&label=Check" 
+            height="20px" 
+            alt="Demos & Examples" 
+        ></a>
```
Comment (jrobinAV) [inline README.md]: Here is the result. 
image

while the badge is for exploring our gallery. It s not a check.

**Review by jrobinAV — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (droidbg) [reply README.md]: I have added 3.8 based on [this](https://github.com/Avaiga/taipy/issues/1934#issuecomment-2397818143) comment from @RymMichaut .
Sure I will add badges for the other version and will remove 3.8.

File: README.md (hunk 2 above)
Comment (droidbg) [reply README.md]: okay I will update this badge to "Explore | Gallery"

File: README.md [hunk 3]
```diff
@@ -30,6 +24,25 @@
         ></a>
 </div>
 <br>
+<div align="center">
+   <img 
+        src="https://img.shields.io/badge/python%20version-3.9-ff371a?style=plastic&labelColor=1f1f1f"
```
Comment (jrobinAV) [inline README.md]: I would use a single badge for all versions.

```suggestion
        src="https://img.shields.io/badge/python-3.9%20%7C%203.10%20%7C%203.11%20%7C%203.12-ff371a?style=plastic&labelColor=1f1f1f"        
```

File: README.md [hunk 4]
```diff
@@ -30,6 +24,25 @@
...
+        alt="Python version needed: 3.10" 
+    />
+    <img 
+        src="https://img.shields.io/badge/python%20version-3.11-ff371a?style=plastic&labelColor=1f1f1f" 
+        alt="Python version needed: 3.11" 
+    />
+        <img 
+        src="https://img.shields.io/badge/python%20version-3.12-ff371a?style=plastic&labelColor=1f1f1f" 
+        alt="Python version needed: 3.12" 
+    />
```
Comment (jrobinAV) [inline README.md]:
```suggestion
```

**Review by jrobinAV — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RymMichaut) [reply README.md]: @droidbg, do you think, we could add all of them in a single card? this would avoid the redundancy of "Python version" 4 times
image

**Comment by RymMichaut** [comment]
and @droidbg could you please recenter everything?

File: README.md (hunk 4 above)
Comment (droidbg) [reply README.md]: I have pushed the fix for above feedback
image

**Review by jrobinAV — DISMISSED** [review]

**Comment by RymMichaut** [comment]
one last thing @droidbg 
could you please add emojis to "Explore docs" and "Explore gallery" ? 
and if it's ok, the PR will be merged

**Review by RymMichaut — DISMISSED** [review]
I'll have to work on this one later

**Review by jrobinAV — APPROVED** [review]

**Pull request merged**